ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/capi/cucushift-installer-rehearse-aws-ipi-capi-workflow.yaml

```diff
@@ -0,0 +1,9 @@
+workflow:
+  as: cucushift-installer-rehearse-aws-ipi-capi
+  steps:
+    pre:
+    - chain: cucushift-installer-rehearse-aws-ipi-capi-provision
+    post:
+    - chain: cucushift-installer-rehearse-aws-ipi-capi-deprovision
+  documentation: |-
+    This is the workflow to trigger Prow's rehearsal test when submitting installer with capi steps/chain/workflow from v-4.11
```

Comment:
How about:
s/ starting v-4.11/. This is starting from 4.11/

Reply:
Done